src/main/java/com/codurance/allaboard/core/actions/topic/FetchTopicById.java

```diff
@@ -0,0 +1,22 @@
+package com.codurance.allaboard.core.actions.topic;
+
+import com.codurance.allaboard.core.model.topic.Topic;
+import com.codurance.allaboard.core.model.topic.Topics;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FetchTopicById {
+
+  private final Topics topics;
+
+  @Autowired
+  public FetchTopicById(Topics topics) {
+    this.topics = topics;
+  }
+
+  public Optional<Topic> execute(Long id) {
```

Comment:
I know why you use execute but we didn't use it anywhere also so think carefully if applying it or not

Reply:
Generally, the idea is to unify the interface underlying all actions and this is the first step towards it. 
Also, the tautology the original code presents, makes the actual code usage difficult, for example, saveSurvey.save(survey);